volumes/miovision/sql/function/function-clear-volumes_15min.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE FUNCTION miovision_api.clear_volumes_15min(
```

Comment:
created!
no specifying for intersection this time?

Reply:
IIRC it's because this function is much faster than the 15min_mvt one so it's not as big a deal to re-run the whole date. 
~2 minutes vs ~20s
![image](https://github.com/CityofToronto/bdit_data-sources/assets/80077912/bdb5487f-a1e6-405b-bb64-1e28f2836526)